android/src/main/java/com/twiliorn/library/RNVideoViewGroup.java

```diff
@@ -107,9 +109,15 @@ protected void onLayout(boolean changed, int l, int t, int r, int b) {
                 videoWidth = 640;
             }
 
+            float aspectRatio = (float) videoWidth / (float) videoHeight;
+            if (isRemote) {
+                // NOTE: this is the aspect ratio the remote video is captured at
```

Comment:
Just a note: We may not be able to guarantee this every time... because experts could have different devices that have different cameras (i.e. experts are on PC laptops) but it's probably a safe bet for average :)